libraries/triedent/src/gc_queue.cpp

```diff
@@ -108,7 +108,7 @@ namespace triedent
 
    void gc_queue::notify_run()
    {
-      std::lock_guard{_queue_mutex};
+      std::lock_guard g{_queue_mutex};
```

Comment:
This changes the behavior slightly. The original code is correct and it is intentional.